roles/openshift_health_checker/library/etcdlogs.py

```diff
@@ -0,0 +1,95 @@
+# pylint: disable=missing-docstring
+
+"""Interface to journalctl"""
+
+from time import time
+import json
+import re
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def exit_json(module, failed, matched, result):
+    module.exit_json(
+        changed=False,
+        failed=failed,
+        matched=matched,
+        result=result,
+    )
+
+
+def stamp_too_old(stamp, time_limit):
+    epoch = int(stamp) / 1000000
+    return epoch - (time() - time_limit) < 0
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            log_count_limit=dict(type="int", default=500),
+            start_match=dict(type="str", default="Starting Etcd Server"),
+            log_matcher=dict(type="dict", required=True),
+            output=dict(type="str", default="json"),
+        ),
+    )
+
+    cmd = [
+        '/bin/journalctl',
+        '-ru', 'etcd',
```

Comment:
The unit? It'll be `etcd` for multi-master clusters or the master unit name which depends on the deployment type